controllers/keda/util/string_lists.go

```diff
@@ -35,3 +39,33 @@ func Remove(list []string, s string) []string {
 	}
 	return list
 }
+
+// Append a new string into a string that has seprator
```

Comment:
```suggestion
// AppendIntoString appends a new string into a string that has seprator
```

In Go, comments on functions should start with the name of the function, please check that also in the rest of the PR.